### PR TITLE
Documentation for changes by #1843

### PR DESCRIPTION
**Pull request by pkess**
@sampsyo is this documentation what you requested in #1843?

### CODE REVIEW

File: docs/guides/advanced.rst [hunk 1]
```diff
@@ -117,7 +117,10 @@ using the :ref:`modify-cmd` command::
 
     beet modify context=party artist:'beastie boys'
 
-And then :doc:`query </reference/query>` your music just as you would with any
+By default beets will show you the changes that are about to be applied and ask
+if you really want to apply them to all, some or none of the items or albums.
+
```
Comment (sampsyo) [inline docs/guides/advanced.rst]: One more sentence of detail on "select" could be useful here. For example:

> You can type y, n, or s for "select." If you choose the latter, the command will prompt you for each individual matching item or album.

File: docs/reference/cli.rst [hunk 2]
```diff
@@ -262,7 +262,8 @@ them. As with other commands, the ``-a`` option matches albums instead of items.
 
 To perform a "dry run", just use the ``-p`` (for "pretend") flag. This will
 show you a list of files that would be moved but won't actually change anything
-on disk.
+on disk. In advance the ``-t`` option sets the timid mode which will ask again
```
Comment (sampsyo) [inline docs/reference/cli.rst]: Delete "in advance"

**Comment by sampsyo** [comment]
Yes; great! Just two small suggestions.

**Comment by pkess** [comment]
Done

**Comment by sampsyo** [comment]
Done!

**Pull request merged**